contracts/FlowIDTableStaking.cdc

```diff
@@ -1822,6 +1822,91 @@ access(all) contract FlowIDTableStaking {
         return true
     }
 
+    /// Validates that a networking address is properly formatted
+    /// Requirements:
+    /// 1. Must not be an IP address
+    /// 2. Must contain a port number after a colon
+    /// 3. Must be a valid domain name format
+    access(contract) view fun isValidNetworkingAddress(address: String): Bool {
```

Comment:
what do you guys think about making this `access(all)` instead `access(contract)` so that any client side apps like FlowPort can use it to verify the networking address before submitting the transaction.
The function does not modify state so should be safe to expose.

Reply:
yes, we should make this `access(all)`

Reply:
changed it to `access(all)`.